### ISSUE TITLE
Tag Not Being Appended

### ISSUE THREAD

**Issue by bramses** [chhoumann/notetweet_obsidian#24]
The automatc tag append function seems to not work for the options: `Post Tweet` and `Post Selected as Tweet`

![Screen Shot 2022-02-09 at 8 14 25 PM](https://user-images.githubusercontent.com/3282661/153324116-71037a08-80bb-4c2b-8b3a-bbcdf1495feb.png)

**Comment by abhishekmadhu** [comment]
Are the tweets getting posted? @bramses

**Comment by abhishekmadhu** [comment]
For me the whole tweeting thing doesn't work at all. What does the following error message mean? 

![image](https://user-images.githubusercontent.com/21152008/153848277-0b0b88f8-3280-498c-978e-dce71779a5cf.png)

**Comment by bramses** [comment]
> Are the tweets getting posted? @bramses

@abhishekmadhu Yeah, the tweets are getting posted. The success modal pops up with the links

**Comment by chhoumann** [comment]
Thanks for reporting this. The tag-appending path has since been fixed and rewritten.

I verified the current release in a live Obsidian instance using both `Post selection as tweet` and the composer. In both cases, closing the success dialog correctly appended the configured `#tweeted` tag to the active note.

I’m closing this as completed. If it still fails with a particular note structure, please open a new issue with a minimal example note and the command used.
